scripts/cd.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+import argparse
+import subprocess
+
+
+def _run_subprocess(
+    args: str,
+    init_message: str,
+    success_message: str,
+    error_message: str,
+    verbose=False,
+):
+    print(init_message)
+    proc = subprocess.run(args, shell=True, capture_output=True)
+
+    process_output = proc.stdout.decode() or proc.stderr.decode()
```

Comment:
this may hide some stderr messages when both stdout and stderr are provided.
Also, a user will only have a feedback of the process once it's finished.

Reply:
I agree, but please keep in mind that this is not a generalized solution for subprocess execution, but just a helper utility for dev purposes. So far it worked just fine with all the scenarios it has to support.

But if you insist I can dope it a bit. What do you say?